### ISSUE TITLE
[discuss] Try to inherit features from origin (Quicksaver/Tab-Groups)

### ISSUE THREAD

**Issue by vaxul** [projectdelphai/panorama-tab-groups#37]
In an ideal world the original tab groups addon (https://github.com/Quicksaver/Tab-Groups) would work. =) But I understand that this isn't possible.

I retest it in [Firefox 56](https://sourceforge.net/projects/portableapps/files/Mozilla%20Firefox%2C%20Portable%20Ed./Mozilla%20Firefox%2C%20Portable%20Edition%2056.0.2/) and it was pleasant to see how feature rich it was.

Have you thought about inherit some of the of the abilities from the old addon? I think about things like:
- multiple possibilities to view at the tab groups (the "free arrange"-mode is what we have now)
- quick access panel  

I know that not all is possible with the current API, but do you have any plans on specific features?

Maybe someone else have more knowledge of what is possible to port?

**Comment by projectdelphai** [comment]
Could you leave some screenshots or some way to easily learn what you're asking about? Easier than setting up firefox 56 with that addon and checking it right now.

**Comment by vaxul** [comment]
I would recommended to first test the legacy addon by yourself if your aren't familiar with the origin of panorama-tab-groups. :-)

Info for other participants. I've linked the portable FF 56 in the first post. A clone of legacy repository can be installed as a temporary addon.

Here some insights:
* by default, clicking the addon icon (optional group title) a popup (quick access panel) is opening
![image](https://user-images.githubusercontent.com/3798226/49751785-9f2f8200-fcae-11e8-808d-bb90bf87f43b.png)
** a click on a group from the list will switch to the group
** optional it's possible to expand a group to list its tabs which allow switching to a specific tab
![image](https://user-images.githubusercontent.com/3798226/49751940-18c77000-fcaf-11e8-96b9-76025887f227.png)
* by clicking "Manage my groups" you will get a familiar looking interface:
![image](https://user-images.githubusercontent.com/3798226/49752074-883d5f80-fcaf-11e8-87e0-5c6dc4b44931.png)
*  But different manage layouts are available:
![image](https://user-images.githubusercontent.com/3798226/49752104-a7d48800-fcaf-11e8-862a-1551c3742c96.png)
![image](https://user-images.githubusercontent.com/3798226/49752121-b622a400-fcaf-11e8-8ef8-519038f186e4.png)
* there are a lot of options:
![image](https://user-images.githubusercontent.com/3798226/49752187-eff3aa80-fcaf-11e8-8b05-5dc3328a2582.png)
** f.e. you could disable the "quick access panel" to get directly to the preferred management interface link in panorama-tab-groups.

The "How to"-section in the addons option menu is really inspiring. ;-)

I read in the docs for web-ext and I think something similar should be possible.

**Comment by projectdelphai** [comment]
I'm familiar with the old add-on, it's just been a while since I've used it and wanted some idea of what you were talking about before I set up an old instance.

The different layouts is something that has been requested but it's a pretty big task so it hasn't been done yet. The extra options are something that we're slowly working through. Generally creating one issue for each feature request is easier to manage so that people can work on each issue individually. But most things should be possible in some sort of way and we'll work through them for sure.
